app/build.gradle

```diff
@@ -14,7 +14,7 @@ android {
         minSdkVersion 21
         targetSdkVersion 29
         versionCode 22
-        versionName '3.0.4'
+        versionName '3.0.5'
```

Comment:
Sorry, no, leave this at '3.0.4'. 
The build script Chris put in will append "pull-545" to the '3.0.4' for an "alpha" build and thus give the apk a unique version name. The CI's PR build can be seen at https://build.palaso.org/project.html?projectId=StoryProducer (log in as guest)
FYI, an app release involves incrementing both versionCode and versionName. I do that when ready to publish a beta. See VersionCode at https://developer.android.com/studio/publish/versioning

Reply:
Got it. Fixed!